<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="label_entityName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelTransform">
     <property name="text">
      <string>&lt;b&gt;Transform&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Translation">
     <item>
      <widget class="QLabel" name="labelTrans">
       <property name="text">
        <string>Translation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="transX">
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="transY">
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Scale">
     <item>
      <widget class="QLabel" name="labelSca">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="scaX">
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="scaY">
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelShapeRenderer">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Shape Renderer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_Shape">
     <item>
      <property name="text">
       <string>Empty</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Circle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Infinite</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fill Color</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ColorLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox_FillColor_R">
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox_FillColor_G">
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox_FillColor_B">
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fill Style</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_FillStyle">
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dotted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cross</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Brush</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stroke Color</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="StrokeLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox_StrokeColor_R"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox_StrokeColor_G"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox_StrokeColor_B"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stroke Thickness</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="spinBox_StrokeThicc"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stroke Style</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_StrokeStyle">
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash - Dot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dash - Dot - Dot</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
